packages/phishing-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.6.0]
+
+### Uncategorized
+
+- Revert "release: 435.0.0 (#5965)" ([#5965](https://github.com/MetaMask/core/pull/5965))
+- release: 435.0.0 ([#5965](https://github.com/MetaMask/core/pull/5965))
+
```

Comment:
These are just development-only changes, so you can leave these out of the changelog:

``` suggestion
```